spec/calendar.mdwn

```diff
@@ -12,22 +12,44 @@ A **Calendar** object has the following properties:
   Any valid CSS colour value. The colour to be used when displaying events associated with the calendar. The colour SHOULD have sufficient contrast to be used as text on a white background.
 - **isVisible**: `Boolean`
   Should the calendar's events be displayed to the user at the moment?
+- **parentId**: `String|null`
```

Comment:
CalDAV theoretically supports nested calendars (because WebDAV supports collection hierarchies) but I haven't seen them and don't know how clients would present them. I don't really see a need, and it can be done with vendor-specific parent/child pointers if really necessary for a specific application.